### PR TITLE
made/rootSaga, make rootSaga for all sagas;

### PR DESCRIPTION
**Pull request by ZelikSV**

### CODE REVIEW

File: src/sagas/user.saga.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { call, put, take, takeEvery } from 'redux-saga/effects';
 import getProfile from 'api/getProfile';
 import { FETCH_USER, GET_USER_CURRENCY_DIALS, SIGNIN_SUCCESS, SIGNUP_SUCCESS } from 'actions/types';
 import { fetchUser, fetchUserSuccess, serverError, fetchDialsSuccess, createDbProfileStart, createDbProfileSuccess } from 'actions';
-import { data, defUser } from '../../mocks/db';
+import { data, defUser } from '../mocks/db';
```
Comment (AlexYalovoy) [inline src/sagas/user.saga.js]: from 'mocks/db'

Comment (Bodyan322) [inline src/sagas/user.saga.js]: remove import from mocks, rewrite on =>
import { getDealsConvertationfromDB } from 'api/database';

Comment (ZelikSV) [reply src/sagas/user.saga.js]: You will do it when merge in dev your work on your branch

Comment (Bodyan322) [reply src/sagas/user.saga.js]: u can just remove this line

Comment (AndrZakhara) [reply src/sagas/user.saga.js]: You must add comment //todo

File: src/sagas/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { all } from 'redux-saga/effects';
+import currencyRootSaga from './currency.saga';
+import { getUserDialsData, watchGetAllUser } from './user.saga';
```
Comment (motifej) [inline src/sagas/index.js]: just `user` saga

**Pull request merged**